src/Action/SearchAction.php

```diff
@@ -114,7 +114,15 @@ public function __invoke(Request $request): Response
                 ];
             }
             $page = (int) $pager->getPage();
-            $total = (int) $pager->getNbResults();
+            if (method_exists($pager, 'countResults')) {
+                $total = (int) $pager->countResults();
```

Comment:
Is it needed to cast ?

Reply:
we can't control the return type hint on `countResults` for now. So it could return whatever technically. So I kept it consistent with casting the return from `getNbResults()`. We can remove the casting on master though.

Reply:
ok

Reply:
IMO, we should also add a `NEXT_MAJOR` comment for the casting.